subscriptions.go

```diff
@@ -0,0 +1,197 @@
+		UnitAmountInCents int      `xml:"unit_amount_in_cents"`
+		Quantity          int      `xml:"quantity,omitempty"`
+	}
+
+	// NewSubscription is used to create new subscriptions.
+	NewSubscription struct {
+		XMLName                 xml.Name             `xml:"subscription"`
+		PlanCode                string               `xml:"plan_code"`
+		Account                 Account              `xml:"account"`
+		SubscriptionAddOns      *[]SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+		CouponCode              string               `xml:"coupon_code,omitempty"`
+		UnitAmountInCents       int                  `xml:"unit_amount_in_cents,omitempty"`
+		Currency                string               `xml:"currency"`
+		Quantity                int                  `xml:"quantity,omitempty"`
+		TrialEndsAt             types.NullTime       `xml:"trial_ends_at,omitempty"`
+		StartsAt                types.NullTime       `xml:"starts_at,omitempty"`
+		TotalBillingCycles      int                  `xml:"total_billing_cycles,omitempty"`
+		FirstRenewalDate        types.NullTime       `xml:"first_renewal_date,omitempty"`
+		CollectionMethod        string               `xml:"collection_method,omitempty"`
+		NetTerms                types.NullInt        `xml:"net_terms,omitempty"`
+		PONumber                string               `xml:"po_number,omitempty"`
+		Bulk                    bool                 `xml:"bulk,omitempty"`
+		TermsAndConditions      string               `xml:"terms_and_conditions,omitempty"`
+		CustomerNotes           string               `xml:"customer_notes,omitempty"`
+		VATReverseChargeNotes   string               `xml:"vat_reverse_charge_notes,omitempty"`
+		BankAccountAuthorizedAt types.NullTime       `xml:"bank_account_authorized_at,omitempty"`
+	}
+
+	// UpdateSubscription is used to update subscriptions
+	UpdateSubscription struct {
+		XMLName            xml.Name             `xml:"subscription"`
+		Timeframe          string               `xml:"timeframe,omitempty"`
+		PlanCode           string               `xml:"plan_code,omitempty"`
+		Quantity           int                  `xml:"quantity,omitempty"`
+		UnitAmountInCents  int                  `xml:"unit_amount_in_cents,omitempty"`
+		CollectionMethod   string               `xml:"collection_method,omitempty"`
+		NetTerms           types.NullInt        `xml:"net_terms,omitempty"`
+		PONumber           string               `xml:"po_number,omitempty"`
+		SubscriptionAddOns *[]SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	}
+
+	// SubscriptionNotes is used to update a subscription's notes.
+	SubscriptionNotes struct {
+		XMLName               xml.Name `xml:"subscription"`
+		TermsAndConditions    string   `xml:"terms_and_conditions,omitempty"`
+		CustomerNotes         string   `xml:"customer_notes,omitempty"`
+		VATReverseChargeNotes string   `xml:"vat_reverse_charge_notes,omitempty"`
+	}
+)
+
+const (
```

Comment:
Fixed in [4687d1](https://github.com/blacklightcms/go-recurly/pull/10/commits/9278698024ab9b18b0b4c965637f58f60a4687d1)